release.sh

```diff
@@ -71,7 +71,14 @@ then
   exit 1
 fi
 
-echo Copying over base Dockerfiles
+printf "\n\nDeleting old container images\n\n"
+
+until docker images | grep "containers.schibsted.io/finntech/node" | awk "{print $3}" | sort | uniq | xargs docker rmi -f
```

Comment:
> docker rmi returns a nonzero exit code if there are no images and that evaluates to true, and that's how this works?

yes

> also, the awk program must be wrapped in single (not double) quotes to not print everything (at least on my mac and an ubuntu server I tested with) for whatever reason

Huh, that's weird. I did have it in single quotes, but did a search and replace to double to match the rest of the script. Didn't test that change... Good catch!

> and finally you can replace the loop construct with xargs' -L1 like this to run it for every line:

The thing is, the script might be unable to remove an image since it has dependants, so we have to skip it and come back to it later. So I do think the loop is still needed. Agreed?